Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -48,65 +47,6 @@ def xorg_recursive_deps
     []
   end
 
-  # @private
-  def homebrew_extra_pkg_config_paths
-    paths = []
-    if x11?
-      libs = xorg_recursive_deps.map(&:lib)
-      shares = xorg_recursive_deps.map(&:share)
-      libs.each do |lib|
-        paths << lib/"pkgconfig"
-      end
-      shares.each do |share|
-        paths << share/"pkgconfig"
-      end
-    end
-    paths
-  end
-
-  def homebrew_extra_aclocal_paths
-    paths = []
-    if x11?
-      shares = xorg_recursive_deps.map(&:share)
-      shares.each do |share|
-        paths << share/"aclocal"
-      end
-    end
-    paths
-  end
-
-  def xorg_include_paths
-    xorg_recursive_deps.map(&:include)
-  end
-
-  def xorg_lib_paths
-    xorg_recursive_deps.map(&:lib)
-  end
-
-  def homebrew_extra_isystem_paths
-    paths = []
-    paths += xorg_include_paths if x11?
-    paths
-  end
-
-  def homebrew_extra_library_paths
-    paths = []
-    paths += xorg_lib_paths if x11?
-    paths
-  end
-
-  def homebrew_extra_cmake_include_paths
-    paths = []
-    paths += xorg_include_paths if x11?
-    paths
-  end
-
-  def homebrew_extra_cmake_library_paths
-    paths = []
-    paths += xorg_lib_paths if x11?
-    paths
-  end
-
   def set_x11_env_if_installed
     ENV.x11 = xorg_installed?
   end
```

Comment:
Do you know whether `set_x11_env_if_installed` and `xorg_installed?` are still needed?

Reply:
> Do you know whether set_x11_env_if_installed and xorg_installed? are still needed?

I don't. I am also not sure about the purpose of these... it just mimicks what Homebrew/brew has on a Mac.

Reply:
I believe you can remove both `set_x11_env_if_installed` and `xorg_installed?`.

Reply:
## set_x11_env_if_installed
Mac has `set_x11_env_if_installed` in `Library/Homebrew/extend/os/mac/extend/ENV/super.rb` and uses `ENV.set_x11_env_if_installed` in `Library/Homebrew/cmd/sh.rb`... Are you sure we can delete it?

## xorg_installed?
Linuxbrew uses `xorg_installed?` in `xorg_recursive_deps` function un `Library/Homebrew/extend/os/linux/extend/ENV/super.rb`. The function is not used anywhere (now), so, I guess, we can delete it.

Reply:
As far as I know, there are no particular environment variables that need to be changed for X11 on Linux, so we can delete `set_x11_env_if_installed`. The empty definition in `extend/ENV/super.rb` is sufficient.